src/test/data/quarkusio/_data/versioned/main/index/quarkus.yaml

```diff
@@ -0,0 +1,2810 @@
+# Generated file. Do not edit
```

Comment:
Did you include this on purpose? It's supposed to be in the sample ZIP already, no?